packages/cli/src/render-solidity/renderWorld.ts

```diff
@@ -1,22 +1,34 @@
-import { renderArguments, renderedSolidityHeader, renderImports } from "./common.js";
-import { RenderWorldOptions } from "./types.js";
+import { renderImports } from "./common.js";
+import { renderArguments, renderedSolidityHeader, renderRelativeImports } from "./common.js";
+import { ImportDatum, RenderWorldOptions } from "./types.js";
 
 export function renderWorld(options: RenderWorldOptions) {
   const { interfaceName, storeImportPath, worldImportPath, imports } = options;
+  const baseImports: ImportDatum[] =
+    interfaceName === "IBaseWorld"
+      ? [
+          { symbol: "IStore", path: `${storeImportPath}IStore.sol` },
+          { symbol: "IWorldKernel", path: `${worldImportPath}interfaces/IWorldKernel.sol` },
+        ]
+      : [
+          {
+            symbol: "IBaseWorld",
+            path: `${worldImportPath}interfaces/IBaseWorld.sol`,
+          },
+        ];
+  const importSymbols = [...baseImports, ...imports].map(({ symbol }) => symbol);
```

Comment:
This makes it such that every "userland" World extends `IBaseWorld` instead of `IStore`/`IWorldKernel`, so that also includes the `CoreModule` methods. Unfortunately the distinction is "implicit" by the name of the interface (`IBaseWorld`), but I felt like adding another option for this to the codegen config would be an overkill.